scripts/smoldyn.py

```diff
@@ -1,57 +1,138 @@
+                        float(camera_defaults["up_vector"]["2"]),
+                    ]
+                )
+            if "look_at_position" in camera_defaults and len(camera_defaults["look_at_position"]) == 3:
+                look_at_position = np.array(
+                    [
+                        float(camera_defaults["look_at_position"]["0"]),
+                        float(camera_defaults["look_at_position"]["1"]),
+                        float(camera_defaults["look_at_position"]["2"]),
+                    ]
+                )
+            if "fov_degrees" in camera_defaults:
+                fov_degrees = float(camera_defaults["fov_degrees"])
+
+            camera_data = CameraData(
+                fov_degrees=fov_degrees,
+                position=position,
+                up_vector=up_vector,
+                look_at_position=look_at_position,
+            )
+
+        if "model_meta_data" in metadata:
+            model_data = metadata["model_meta_data"]
+            model_meta_data = (
+                ModelMetaData(
+                    title=model_data.get("title"),
+                    version=model_data.get("version", ""),
+                    authors=model_data.get("author", ""),
+                    description=model_data.get("description", ""),
+                    doi=model_data.get("doi", ""),
+                    source_code_url=model_data.get("source_code_url", ""),
+                    source_code_license_url=model_data.get(
+                        "source_code_license_url", ""
+                    ),
+                    input_data_url=model_data.get("input_data_url", ""),
+                    raw_output_data_url=model_data.get("raw_output_data_url", ""),
                 ),
-                doi="10.1016/j.bpj.2016.02.002",
-                source_code_url="https://github.com/allen-cell-animated/simulariumio",
-                source_code_license_url="https://github.com/allen-cell-animated/simulariumio/blob/main/LICENSE",
-                input_data_url="https://allencell.org/path/to/native/engine/input/files",
-                raw_output_data_url="https://allencell.org/path/to/native/engine/output/files",
-            ),
+            )
+
+    # spatial units defaults to 1.0 meters
+    spatial_units = UnitData(
+        name=event["spatial_units"]["name"]
+            if "spatial_units" in event and "name" in event["spatial_units"]
+            else "meter",
+        magnitude=float(event["spatial_units"]["magnitude"])
+            if "spatial_units" in event and "magnitude" in event["spatial_units"]
+            else 1.0,
+    )
+
+    # time units default to 1.0 seconds
+    time_units = UnitData(
+        name=event["time_units"]["name"]
+            if "time_units" in event and "name" in event["time_units"]
+            else "second",
+        magnitude=float(event["time_units"]["magnitude"])
+            if "time_units" in event and "magnitude" in event["time_units"]
+            else 1.0,
```

Comment:
can you pull this from simulariumio? so it's getting the true default and not hardcoded here.